csep/utils/readers.py

```diff
@@ -473,8 +473,69 @@ def parse_datetime(dt_string):
         else:
             return events, catalog_id
 
-def read_em_gcmt(fname):
-    raise NotImplementedError("not implemented yet!")
+
+
+def read_ingv_rcmt_csv(fname):
+    """
+    Reader for the INGV (Istituto Nazionale di Geofisica e Vulcanologia - Italy)  European-
+    Mediterranean regional Centroid Moment Tensor Catalog.
+    It reads a catalog in .csv format, directly downloaded from http://rcmt2.bo.ingv.it/ using the Catalog Search (Beta
+    version).
+
+    The ZMAP Format has the following dtype:
+
+    dtype = numpy.dtype([('longitude', numpy.float32),
+                        ('latitude', numpy.float32),
+                        ('year', numpy.int32),
+                        ('month', numpy.int32),
+                        ('day', numpy.int32),
+                        ('magnitude', numpy.float32),
+                        ('depth', numpy.float32),
+                        ('hour', numpy.int32),
+                        ('minute', numpy.int32),
+                        ('second', numpy.int32)])
+    """
+
+    import time             ### Should these modules be imported in top level of readers.py?
+    import pandas
+
+    start_time = time.time()
```

Comment:
i dont think we should time the io operation inside this function, and the timing block never ends (i dont see an end_time or print statement with the elapsed time). we can remove this line.

Reply:
ooops mb

Reply:
😛